src/commands/faulenzerping.ts

```diff
@@ -0,0 +1,90 @@
+import type { Client, Message, Snowflake } from "discord.js";
+
+import { BotContext } from "../context.js";
+import { MessageCommand } from "./command.js";
+import type { ProcessableMessage } from "../handler/cmdHandler.js";
+import { isTrusted } from "../utils/userUtils.js";
+import { chunkArray } from "../utils/arrayUtils.js";
+
+export class FaulenzerPingCommand implements MessageCommand {
+    name = "faulenzerping";
+    description = "Pingt alle Leute, die noch nicht auf die ausgewählte Nachricht reagiert haben, aber in der angegebenen Gruppe sind.";
+
+    async handleMessage(message: ProcessableMessage, _client: Client<boolean>, context: BotContext): Promise<void> {
+        if (!isTrusted(message.member)) {
+            await message.reply("Du bist nicht berechtigt, diesen Command zu benutzen.");
+            return;
+        }
+
+        if (message.reference?.messageId === undefined) {
+            await message.reply("Brudi du hast kein Reply benutzt");
+            return;
+        }
+        const messageThatWasRepliedTo = await message.fetchReference();
+
+        const { ignoredRoleIds, maxNumberOfPings, minRequiredReactions } = context.commandConfig.faulenzerPing;
+
+        const roles = [...message.mentions.roles.filter(role => !ignoredRoleIds.has(role.id)).values()];
```

Comment:
Wenn ich eine Rolle pinge, die ich eigentlich nicht pingen darf - zählt das auch als Mention? We will see..

Reply:
Ich hab keine Ahnung, der gesamte Code ist auch ungetestet